spec/lib/import_polygons_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'rails_helper'
+require 'import_polygons'
```

Comment:
I don't understand why this is needed?

Reply:
Most of our lib specs have a require line or they don't load the class they're testing. I guess this is because lib/ is not in app/. Could do something like the existing `Dir[Rails.root.join('spec/support/**/*.rb')].each { |f| require f }` which is in the rails_helper.rb.

Reply:
I'll make that a separate tech debt ticket

Reply:
https://github.com/DFE-Digital/teacher-vacancy-service/pull/1844/files